packages/client/ui/Dialog/DialogContent.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import * as RadixDialog from '@radix-ui/react-dialog'
+import {twMerge} from 'tailwind-merge'
+
+export const DialogContent = React.forwardRef<
+  HTMLDivElement,
```

Comment:
+1 this is technically not correct, as if someone uses `asChild` it could render a different html element  - we should use `React.ElementRef<typeof DialogContent.Title>`. not sure if this type is available in React 17 though, not a blocker though

Reply:
There is no such type `ElementRef`, I wasn't sure what to put here instead